src/styles/layout.scss

```diff
@@ -11,6 +12,7 @@ html, body, #root {
   display: flex;
   align-items: center;
   padding-top: 40px;
+  padding-bottom: $navbar-height;
```

Comment:
Isn't the `$navbar-height` the height of the very top navbar? (And not the bottom navbar.)

Reply:
@tomclose Andrew is referring to the footer height here, which is the same as the navbar height